src/frontend/shortlist/src/App.vue

```diff
@@ -10,6 +10,22 @@ const apiClient = new ShortlistApi("https://api.shortlist.nyc");
 const router = useRouter();
 const appSessionStore = sessionStore();
 
+function moveAcceptedSchoolCardToDB(payload) {
+  console.log(payload);
+  let apiReq = {
+    recoID: payload.recoID,
+    accepted: payload.accepted,
+  };
+  let success = () => {
+    console.log("Added to a list successfully");
```

Comment:
This doesn't actually add to the list, only marks the recommendation as accepted. Same for error message below

Reply:
What is the error message you were referring to? Anything from the console?